src/views/settings/SettingsAbout.tsx

```diff
@@ -202,7 +202,7 @@ const SettingsAbout: Screen<"SettingsAbout"> = ({ navigation }) => {
           </NativeText>
           {PackageJSON.dependencies["react-native"]  &&
               <NativeText variant="subtitle">
-                RN : {PackageJSON.dependencies["react-native"].split("^")[1]} | Expo : {(PackageJSON.devDependencies.expo || PackageJSON.dependencies.expo).split("^")[1]}
+                RN : {PackageJSON.dependencies["react-native"].split("^")[1]} | Expo : {(PackageJSON.devDependencies["expo"] || PackageJSON.dependencies["expo"]).split("~")[1]}
```

Comment:
et le code à indenter en plus :

```suggestion
                RN : {PackageJSON.dependencies["react-native"].split("^")[1]} |
                Expo :{" "}
                {(
                  PackageJSON.devDependencies.expo ||
                  PackageJSON.dependencies.expo
                ).replace("^", "").replace("~", "")}
```